src/rust/plugin-work-queue/migrations/1640037910_plugin_registry_migration.sql

```diff
@@ -0,0 +1,86 @@
+CREATE TYPE status_t AS ENUM ('enqueued', 'failed', 'processed');
+
+CREATE SCHEMA plugin_work_queue;
+
+CREATE FUNCTION megabytes(bytes integer) RETURNS integer AS
+$$
+BEGIN
+    RETURN bytes * 1000 * 1000;
+END;
+$$ LANGUAGE plpgsql;
```

Comment:
1. can you document what the 256MB restriction is for?
2. one could argue its 1024 just saying

Reply:
It seems like a reasonable restriction on a given binary. We can always raise it without issue. I just don't want someone shoving 50GB into the database.